getter.go

```diff
@@ -11,8 +11,8 @@ type Key string
 // Getter is a map of template keys to Template that allows looking up a
 // given template by key.
 type Getter struct {
-	m            map[Key]*Template
-	sync.RWMutex // RWMutex protects m.
+	m    map[Key]*Template
+	lock sync.RWMutex // RWMutex protects m.
```

Comment:
Thanks @aeijdenberg ! Can you please rename `lock` to `mu` and change comment to `mu protects m.`?

Former removes stutter too: `ts.lock.RLock()` --> `ts.mu.RLock()`.

Aside: I just see now after my rename I forgot to rename the receiver to `g` (it's still `ts).